src/main/java/org/apache/sysds/runtime/privacy/propagation/OperatorType.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.sysds.runtime.privacy.propagation;
+
+public enum OperatorType {
```

Comment:
this could be considered to be moved into common/Types.java
or leverage already existing Operator Types?

Reply:
Right now, it is only used for privacy propagation, hence it is not a common type.

Reply:
The existing operator types are not relevant, since I only deal with two types: aggregate and non-aggregate.

Reply:
okay, i do the same in compression. It was just if they matched up completely, then they could be reused, and furthermore used as a TODO list, for what is not supported.